Packs/Core/Playbooks/playbook-A_Successful_login_from_TOR.yml

```diff
@@ -0,0 +1,802 @@
+          accessor: id
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1640
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "14":
+    id: "14"
+    taskid: a5cfd092-601a-4ff2-8f25-7cabd460ec84
+    type: title
+    task:
+      id: a5cfd092-601a-4ff2-8f25-7cabd460ec84
+      version: -1
+      name: Done
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 1800
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "15":
+    id: "15"
+    taskid: a6e46587-a13a-4ca9-8276-199a1743d113
+    type: regular
+    task:
+      id: a6e46587-a13a-4ca9-8276-199a1743d113
+      version: -1
+      name: Expire User Password With Active Directory
+      description: Expires the password of an Active Directory user.
+      script: '|||ad-expire-password'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "8"
+    scriptarguments:
+      username:
+        simple: ${Core.OriginalAlert.event.login_data_dst_normalized_user.username}
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -460,
+          "y": 990
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "16":
+    id: "16"
+    taskid: 82ced8bc-aea8-486b-8f86-374b6254bc37
+    type: regular
+    task:
+      id: 82ced8bc-aea8-486b-8f86-374b6254bc37
+      version: -1
+      name: Get User Identity Details
+      description: Returns information about each alert ID.
+      script: '|||core-get-cloud-original-alerts'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    scriptarguments:
+      alert_ids:
+        simple: ${alert.id}
+      filter_alert_fields:
+        simple: "false"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 200
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "19":
+    id: "19"
+    taskid: f0c501d7-19d6-4ef6-8864-980df56d8132
+    type: regular
+    task:
+      id: f0c501d7-19d6-4ef6-8864-980df56d8132
+      version: -1
+      name: MS-Graph Azure AD - Check If the User Exists
```

Comment:
i would change that to just Azure AD - Check if the user exists

Reply:
and again, this task does not check if the user exist, it collects the the user, in task #20 you check if the user exists, and you disable it.
so the names of the tasks should be
#19 - Azure AD - search user
#20 - Check if the user exists
#22 - Azure AD - Disable User